### ISSUE TITLE
[RFC] much needed gt++ multiblock balance 

### ISSUE THREAD

**Issue by 0lafe** [GTNewHorizons/NewHorizons#3701]
#### Which modpack version are you using?
2.0.5.1
#

#### What do you suggest instead/what changes do you propose?

GT++ multiblocks are simply dump OP compared to single blocks of any tier. They're also better than using the best possible processing array in most cases. Their upgrade cost is limited to the energy hatch, which even with a Nerf will still most likely be less than a machine cost. However this means making them is almost the same in any tier you're in, and means you can keep your multiblock with only a small upgrade fee. Some multiblocks like the ebf/vac freezer have an associated cost with them, which is a nice way to balance them a little, but most run the exact same as their gt counterparts, except faster, cheaper, and available earlier in many cases. There really can't be any arguement around them being better, they simply are. 

Now, do we need better machines? No, not really. The PA is perfectly fine up past tiers that GT++ is concerned about, and out preformes a few of the base GT multiblocks. However it is the most expensive multiblock to upgrade as it'll always cost 64 machines of the next tier. This is a good thing, it's the perfect cost. The machine can only go as fast as the machines you can make for it. Much of LV-EV is making single blocks which all follow a trend of 4x power for 2x speed. It's a good rule, you get some fast machines eventually but it'll cost you a good amount of power. However the GT++ multiblocks come in at EV-LuV and give you speed bonuses of up to 6x when compared to a single blocks of the same tier. The multiblock is a multiblock, but the cost is usually not that much more than a single block. The controller is usually about the same as a single block, while the casings just cost 8 ingots of what is essentially a tier appropriate metal, that's just made in the abf instead of the ebf. 

Complaining asside, my suggestion is that they should be spaced out a LOT more. Each multiblock is a significant improvement over the single block and array, there's no reason you should get them all in about 2 tiers when there's still a bunch of tiers left in the game. IV+ doesn't really have much to do, part of that is because there needs to be more different content available at those tiers, but part is because EV really goes all out with a lot of stuff. Endgame thaum, nukes, fluid nukes, and a good amount of the gt++ multiblocks. Spread them out a little, have maybe 2-3 per tier, with the better ones higher up. My current suggestion is this for tiering 

EV: ore washer, thermal centrifuge, packager, electrolyzer (slow machines that I honestly skipped a lot just because I felt like the singles were too slow for a while)
IV: cutter, centrifuge, arc furnace (pending), large steam turbines(centrifuge is a little better than the EV ore machines for being able to do bee Combs too, arc furnace for annealed copper for circuits, turbines for fluid reactors/solar thingy)
LuV: wiremill, extruder, maceration stack(lots of wires and extruded parts for the assembly line stuff you start making, centrifuge higher than other ore stuff because it also does bee things, quarry in this tier so maceration stack for solid ores again) 
ZPM: material press, multimachine (useful machines, multi machine for lasers for SoCs and for doing better Combs+crops) 
UV: vac freezer, ebf (now that you're in UV it's mostly fusion time, so it's fitting to give that one the benefits of better ebfing)
UHV: assembler, circuit assembler (no single blocks after UHV so good to finish single block ones at UHV, those are probably the best 2 so unlocked last)
UEV: void miner if that comes out, lategame but around the time you'd start needing it.
UIV: better assembly line? ;D maybe? One day? We can hope 
I could provide some recipes, but the main thing is the tiering. ZPM could be the cost of a single block, or something with the assembly line, I think the important part to figure out first is the tiers you unlock them. 

Buffs: I think if this happens and the multiblocks are made to go all the tiers single blocks do, they should be able to do recipes faster than 1/s. They already seem to do that but from what alk said that seems like a glitch that I don't mind not getting patched. 

Nerfs pt2: in addition to tiering requirements, I think it would be good for the multis themselves to be much more expensive. Part of building a PA is the huge increased cost, but that's good for the later tiers. You have more infrastructure, more ores, faster machines, you should be expected to build bigger machines. Each casing costing 8 ingots is kind of boring, what if the recipe was like 1296L of the material fluid in an assembler with some circuits/superconductors of the tier you're in. Something that's easier than just cooking a lot of ingots. By that time you have AE, PR pipes, SFM, OC and all the other inventory sorting options in the pack. Part of a hard pack imo is that it forces you to grow if you want to progress. You have all these tools available to you in these later tiers, you should be forced to use some combination of them to achieve a goal. 

TL;DR 

GT++ multis OP, tier them across the tiers we got instead of clustering them at EV-LuV, make the pack harder in those tiers. Also harder recipes for casings maybe? Plz? 

Uh, forgot to mention that it kinda fits with extending normal GT multiblocks. At HV you get distillery, LCR and the oil cracker, which while not quite like GT++ give similarly useful bonuses over the single blocks.

**Comment by richardhendricks** [comment]
Sifter has no increased output.

**Comment by richardhendricks** [comment]
I think the sifter should be HV, to help players get chrome once they get to that tier.  Everything else looks fine to me.

I think overall pollution for all blocks should be increased across the board, with MBs unlocked at higher tier increasing more. GT++ multis requiring mufflers is a great first step.

**Comment by richardhendricks** [comment]
The electric blast furnace, pyrolyze, and power sub station are the only mutlis that have unique upgrades.  I do like the idea, like upgradeable teeth/blades/crushing assembly in the macerator, etc.

**Comment by 0lafe** [comment]
>Maceration Stack (not ZPM) -> LuV .. so you just did centrifuge in IV and then it would be nice to look forward to the daddy macerator in LuV. By LuV players should be able to get a nice and full processing line set up then start focusing on machines in ZPM+

Moved 

>Packager -> IV ... got to let people struggle with sorting them dusts for a little while!!! I think LuV is unnecessary as it is now.

I mean, this is the struggle cereal of modpack 

>Large Scale Assembler (not UHV) -> UV. I don't see why this interesting stuff needs to be completely unreachable by most semi-hardcore players.

Trying to tier them so for the most part they appear when something should be less relevant. Who wants to spend hours wiremilling for assembly line crafts, or lasering SoC chips. However assembler recipes are really well tiered imo, and you shouldn't get a bonus on them for a while. I see them as being less relevant when there aren't any left, which means it needs a UHV assembler to craft. Personally I'd rather see more recipes in the assembly line for higher tiers, like most LuV+ stuff, then the assembler could be UV for sure. The circuit assembling feature imo shouldn't be available until you've made all the circuits. I'd rather see it as a reward for that. I'd like to see UV assembler multi UHV circuit assembler multi if that's possible

>Just a random thought but is there any ideas to 'upgrade' multiblocks to enable higher tier recipes upgrades etc. Maybe a block you can add to the multiblock that upgrades it (similar to a quarry). I know nothing like that currently exists so its just a high level thought at the moment to help deal with some tier problems.

I.E.: The Extruder cannot process infinity plates unless it has 'Uber UV Hatch' in it's multiblock structure that needs UV specific materials to make so that way we can still get the Extruder earlier without breaking recipes.

Alk's mentioned adding a hatch that needs some circuitey thing to run, which is a cool idea. I'm all for more cost, but I'd rather see better tiering in the multis themselves than just a larger upgrade cost. In my opinion, the multis are the most OP when you have it working on recipes of the same tier as the power because of their straight speed boost. If the tiering lowered their speed (EV Uber hatch thing does 20% IV does 40% LuV does 60%) I'd be for it as the only fix, but otherwise I like Uber hatch + strict tier reqs (maybe throw some lutetium/Ti in the EV, naq in IV, europium in LuV, neutronium in ZPM, Tritanium in UV and figure out a good tier material for UHV and use that).

 There's 2 issues here I guess. Gating the multis and the cost of the multis. Imo the cost for the ZPM+ one's should outweigh the assembly line and be on par with fusion and the such, make it actually difficult to get. Not just ingots for casings, more high tier materials in recipes, stuff like that.

**Comment by richardhendricks** [comment]
There are quite a few multis that are inherently tiered, but I think @draknyte1 said he was running out of meta IDs?

EBF - coils
pyrolyze - coils
oil drill - upgrade whole machine
ore drill - upgrade whole machine
boilers - completely different machine
Power sub station - Upgrade vanadium redox cells

It would be nice if the upgrades could be relevant for the machine.  Like better sifting grids for the sifter that go from fine steel wire -> carbon fiber -> carbon nanotubes -> magic substance.

**Comment by draknyte1** [comment]
Already said no.

**Comment by Ghengis88** [comment]
no to all of this? @draknyte1

**Comment by Anarack** [comment]
Doesn't the DD portal need UV tier components/machines to make? Mainly the components for the material compressor thing needs a bunch of uv parts and you need that no matter what rout you take to do it.

**Comment by 0lafe** [comment]
Ahhh that little thingy. Yes and kinda. It only takes pistons/motors which while being a part of early UV, in no way signifies being fully in the tier. Circuit assemblers/circuits really show the tier you're in. If you can make/power a UV circuit assmbler and make wetwares, you're in UV. Sine wetwares aren't required for UV anymore it's a little less of a gate than it was, but honestly circuits past the crystals need to be look at anyway. So much of their recipes are just, no good.

**Comment by draknyte1** [comment]
You're joking if you think I'd push my Multis any later than LuV/ZPM.

**Comment by 0lafe** [comment]
>Here's an alternative; Don't base your suggestions against the PA and instead, remove the PA.

I'm not basing it on the PA, I'm basing it on you getting like 12 upgrades to single blocks in 2 tiers, when there's still 4 more tiers to the game after that. I'm not basing anything on the PA, I'm just saying you don't need those speeds, nor should you get them as early as you do, especially not stuff like the assembler, or adv ebf. They're clearly dump OP, and that probably won't change, but at least tier them well. No one should get 6x bending speeds at IV. The recipes are balanced to be done at 1x speed+overclocks and stuff, which is good. There's no recipes that need the GT++ multis, everything is more than doable with basegame GT, GT++ just makes it easier for no reason. If you get like a general 2-3x speed boost in IV, that's a HUGE difference in processing time, and you need to make nowhere near the infrastructure, and all the multis cost almost nothing for those tiers. 

>You're joking if you think I'd push my Multis any later than LuV/ZPM.

Give me one good reason they shouldn't. They're OP, clearly, if you want them in the game they should be tiered accordingly.

**Comment by 0lafe** [comment]
>I'd much rather have unique multiblocks than cloning 64 singleblocks for PAs.

Yea that's a cool thing. The problem is when they cost almost a single black of like EV/IV. As you get AE, and amps of IV, crafting stuff is easier than in MV, so a tiered controller that almost looks like a single black craft, and a bunch of ingots, is much easier than making your first ebf. Why is this a problem? Because the difficultly drastically drops off around IV/LuV, partly because you don't have to make multiblocks/single blocks anymore. Hatch cost increase will help, but there's no reason the multi itself should be so cheap. EV multi should take more than ingots for the casings maybe like 4 Ti pipes + 4x HV superconductor + 1296L of the GT++ alloy, in an EV assembler. Say you need 32 casings, it'll cost the same amount of alloy but also 2 stacks HV superconductors and and a bunch of titanium. Which is a good challenge for EV.

**Comment by Highbeam** [comment]
PA’s are nice but they are far from the answer... I hate that the PA process begins whether you have 1 dust in to 64 basically making 63 machines useless until it’s done 1 process. I do agree some recipes could be updated to better tier the gt++ Multis.

Actually making anything higher than LuV/ZPM is extremely questionable. I like that you want to make UV+ interesting but it’s insane to get to get to those tiers without getting some of these multi setups first. By UV you are looking at Assline/fusion.

I think fixing the recipes by adding circuits or something would be fine.

**Comment by WarlordWossman** [comment]
>PA’s are nice but they are far from the answer... I hate that the PA process begins whether you have 1 dust in to 64 basically making 63 machines useless until it’s done 1 process. I do agree some recipes could be updated to better tier the gt++ Multis.

Can you not read with a comparator from the input bus and use a machine controller cover to only start if you reach the wanted amount or above for 1 process to prevent that?

**Comment by Highbeam** [comment]
Yea wossman; or use super buffers...   But I disagree with them as an alternative/answer to anything here. 
 Just because some people struggled with PAs previously doesn't mean we should all suffer and put these multis to insanely high tiers now they are here.  We can and will still use them I'm sure in a new tier reorientation.

I would agree the multiblock recipes could be made harder (not grindier...).

I would also agree with switching the tiers around.

However, I disagree with using these multis to make UV+ interesting...  These are just better versions of machines we already have...  For UV+ consider adding something new and interesting.  I'm personally not playing this pack to UHV+ "just to get a nice assembling multiblock"...  I'd rather play that far to get something game changing!

And finally I personally disagree with nerfing multis just to make them comparable to a PA without some numbers/rational behind it...  If the multis are harder to craft, then the benefits they bring should remain.

As for the 'broken recipes'... Adjust them by adding items relevant to the tiers.

**Comment by 0lafe** [comment]
>However, I disagree with using these multis to make UV+ interesting... These are just better versions of machines we already have... For UV+ consider adding something new and interesting. I'm personally not playing this pack to UHV+ "just to get a nice assembling multiblock"... I'd rather play that far to get something game changing!

That's like saying I don't want to play to EV just to make easier LV circuits, I want them at LV and better stuff at EV. If you don't want to play past UHV that's fine, if you don't want to play past IV that's fine, but I don't get how that means the thing you want should be in an earlier tier. Things are gated for a reason, not just because the lategame needs better content. You don't need better LV circuits in LV, you don't need a 6x speed bender in EV. The new tiers give them placements that make sense, but it doesn't stop anyone from progressing. The GT++ multis are irrelevant from progression, the gameplay is fine without them, there's no need for them in the early tiers. They're significant upgrades. 2x faster ebf recipes is gamechaging, that's 2x less ebfs you need to power. Overall you use like 2-3x less power for stuff which I don't see any way to look at it besides gamechanging. Imagine if making like 10 EV and IV multis boosted your lategame power by 2-3x. 

>And finally I personally disagree with nerfing multis just to make them comparable to a PA without some numbers/rational behind it... If the multis are harder to craft, then the benefits they bring should remain.

Uh yea, that's sort of the idea. Except not a single multi is close to that point yet of being correctly expensive. They're pretty much all cheaper than PAs by a wide margin, but this has almost nothing to do with PAs really. It has everything to do with lowering your power demand, increasing your crafting speeds, and doing it cheaper and easier than normal GT. It's more that GT++ multis are overpowered compared to GT itself than compared to the PA. 

>I would agree the multiblock recipes could be made harder (not grindier...).

>I would also agree with switching the tiers around.

So you agree with the tiers I put there except for the assembler? It's easily the most OP, and if it's the last one that's unlocked then it ends single blocks. There's no reason to do that in gtnh when we have another tier of single blocks past UV. Its also just generally really good, considering how many things are assembler recipes. What's wrong with putting it in UHV besides you don't want to have to play until UHV?

**Comment by draknyte1** [comment]
I’ll say this; you’ll never remove them, because #vote wouldn’t ever majority say yes. 

Be reasonable on recipe costs; I’ll review them. Just don’t expect them to be pushed to tiers 90% of players won’t ever reach, cause that’s stupid.
